### DISCUSSION TITLE
deep-learning/linear-model/minimise-loss-function

### DISCUSSION THREAD

**Discussion by giscus** [luweizheng/lulaoshi.info#84] [Announcements]
# deep-learning/linear-model/minimise-loss-function

线性回归的求解：矩阵方程和梯度下降、数学推导及NumPy实现

https://lulaoshi.info/deep-learning/linear-model/minimise-loss-function.html

**Comment by iceqing** [comment]
写的很详细，易懂，感谢

**Comment by ChengchengCai006** [comment]
写得很好！！！受益匪浅！！！
